client/my-sites/plan-features/index.jsx

```diff
@@ -252,8 +261,9 @@ class PlanFeatures extends Component {
 			const { rawPrice, discountPrice } = properties;
 			const classes = classNames( 'plan-features__table-item', 'has-border-top' );
 			let audience = planConstantObj.getAudience();
+			let billingTimeFrame = planConstantObj.getBillingTimeFrame();
 
-			if ( isInSignupTest ) {
+			if ( isInSignupTest && ! displayJetpackPlans ) {
```

Comment:
This may be a good start, but I'd like to propose a more robust improvement at the [source of the problem](https://github.com/Automattic/wp-calypso/blob/21abe769006a1b7c1509375c81f716f942177f5b/client/my-sites/plan-features/index.jsx#L542-L553)

```js
planFeatures = getPlanFeaturesObject( planConstantObj.getBlogSignupFeatures( abtest ) );
// becomes
if ( planConstantObj.getBlogSignupFeatures ) {
  planFeatures = getPlanFeaturesObject( planConstantObj.getBlogSignupFeatures( abtest ) );
}
```

Same for all `getBlogSignupFeatures`, `getPortfolioSignupFeatures`, and `getSignupFeatures`.